pkg/daemon/ceph/client/pool_test.go

```diff
@@ -572,7 +573,10 @@ func testCreatePoolWithReplicasPerFailureDomain(t *testing.T, failureDomain, cru
 			assert.Equal(t, args[4], "size")
 			poolSize, err := strconv.Atoi(args[5])
 			assert.NoError(t, err)
-			assert.Equal(t, uint(poolSize), poolSpec.Replicated.Size)
+			if uint(poolSize) < math.MaxUint {
```

Comment:
The test will always have pool size smaller than the max, no need for this check
```suggestion
```

Reply:
applied